js/index.js

```diff
@@ -8,35 +8,120 @@ const siteContent = {
+    "services-h4": "Services",
+    "services-content":
+      "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "product-h4": "Product",
+    "product-content":
+      "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4": "Vision",
+    "vision-content":
+      "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  contact: {
+    "contact-h4": "Contact",
+    address: "123 Way 456 Street Somewhere, USA",
+    phone: "1 (888) 888-8888",
+    email: "sales@greatidea.io"
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById("logo-img");
+logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+//nav bar links
+const nav = document.querySelectorAll("header nav a");
+
+nav.forEach(function(value, i) {
+  value.textContent = siteContent.nav[`nav-item-${i + 1}`];
+
+  //adding color to nav links
+  nav.forEach(value => {
+    value.style.color = "green";
+  });
+});
+
+// //Add additional link to end of nav with appendChild()
+//different way to append
+// const newATag = document.createElement('a');
+//      newATag.textContent = 'Rockets'
+//      const secondaryContent = document.querySelector('nav');
+//      secondaryContent.append(newATag)
+//      newATag.style.color = 'green';
+
+//Add additional links to beginning of nav with prepend()
+
+//creating new elements
+const newLink1 = document.createElement("a");
+newLink1.textContent = "home";
+newLink1.style.color = "green";
+
+const newLink2 = document.createElement("a");
+newLink2.textContent = "login";
+newLink2.style.color = "green";
+
+//adding new elements to nav
+const navUpdate = document.querySelector("nav");
+
+navUpdate.prepend(newLink1);
+navUpdate.appendChild(newLink2);
+//navUpdate.style.color = 'green';
+
+//DOM is awesome
+const awesome = document.querySelector(".cta h1");
+awesome.textContent = siteContent["cta"]["h1"];
+
+// //Large circle image of code
```

Comment:
Is the code from here down supposed to be commented out? Something looks a little strange here.